harvard-lts/etd_alma_monitor_service
# Etd 301

**Query mongo for proquest ids.**
* * *

**JIRA Ticket**: [(link)](https://at-harvard.atlassian.net/browse/ETD-301

# What does this Pull Request do?
Queries mongo for proquest ids, returns a list of dicts containing pqids This will be used to determine which, if any pqids to query alma for, to see if they are ready for sending do dims/drs 

# How should this be tested?

A description of what steps someone could take to:
- download branch
- copy the .env.example to .env and populate with dev vars
- bring up container locally (see readme)
- exec into container (see readme)
- run `pytest`(tests should pass)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Y
- integration tests? Y

# Interested parties
Tag (@ mention) interested parties
